### ISSUE TITLE
stale apache PID files can prevent process from starting

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#3553]
In the event of a hard shutdown or power failure, stale Apache PID files are left in place.

Although that doesn't cause a problem for systemd, Apache itself is not starting because its checking that PID file and saying its running even though the PID inside it doesn't itself exist.

**Comment by julsemaan** [comment]
fixed by 7e4609a04e7bb6ac44438aec0a1a01cdb41d83a9

**Comment by julsemaan** [comment]
Fixed unit files can be fetched using:
```
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.aaa.service > /usr/lib/systemd/system/packetfence-httpd.aaa.service
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.admin.service > /usr/lib/systemd/system/packetfence-httpd.admin.service
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.collector.service > /usr/lib/systemd/system/packetfence-httpd.collector.service
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.parking.service > /usr/lib/systemd/system/packetfence-httpd.parking.service
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.portal.service > /usr/lib/systemd/system/packetfence-httpd.portal.service
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.proxy.service > /usr/lib/systemd/system/packetfence-httpd.proxy.service
curl https://raw.githubusercontent.com/inverse-inc/packetfence/ba87e284e805692796bc8366a6b84604e260235f/conf/systemd/packetfence-httpd.webservices.service > /usr/lib/systemd/system/packetfence-httpd.webservices.service

systemctl daemon-reload
```
